src/app/pages/request-query/request-query.component.html

```diff
@@ -16,43 +16,57 @@
 
 SPDX-License-Identifier: Apache-2.0
 -->
-<div *ngIf="dataRequest" class="container">
-  <div class="main-row hero">
-    <h1>Request Query</h1>
-  </div>
-  <div class="mdm-request-query__back-link">
-    <a [routerLink]="[backRouterLink, backRouterRequestId]">
-      <span class="fa-solid fa-arrow-left"></span>
-      {{ backLabel }}
-    </a>
-    <button
-      *ngIf="status === 'ready'"
-      mat-flat-button
-      color="primary"
-      [disabled]="!dirty"
-      (click)="save()"
-    >
-      Save changes
-    </button>
-  </div>
-  <div *ngIf="status === 'loading'" class="main-row mdm-request-query__loading">
-    <mat-spinner color="primary"></mat-spinner>
-    <p>Finding your query...</p>
-  </div>
-  <div *ngIf="status === 'error'" class="highlight-box mdm-request-query__error">
-    Unfortunately there was a problem with your request or query. Please try again.
-  </div>
-  <div *ngIf="status === 'ready'" class="row">
-    <div class="mdm-request-query__header">
-      <h2>
-        {{ dataRequest.label }}
-        <mdm-request-status-chip [status]="dataRequest.status"></mdm-request-status-chip>
-      </h2>
+<div class="container">
+  <div *ngIf="dataRequest">
+    <div class="main-row hero">
+      <h1>{{ dataRequest.label }}</h1>
+    </div>
+    <div *ngIf="status === 'loading'" class="mdm-my-request__progress">
+      <mat-progress-bar color="primary" mode="indeterminate"></mat-progress-bar>
+    </div>
+    <div class="mdm-request-query__back-link">
+      <a [routerLink]="[backRouterLink, backRouterRequestId]">
+        <span class="fa-solid fa-arrow-left"></span>
+        {{ backLabel }}
+      </a>
+    </div>
+
+    <div *ngIf="status === 'error'" class="highlight-box mdm-request-query__error">
+      Unfortunately there was a problem with your request or query. Please try again.
+    </div>
+
+    <div class="col">
+      <div class="mdm-request-query__header">
+        <div class="mdm-request-query__header-title">
+          <div class="mdm-request-query__header-buttons">
+            <button
+              *ngIf="status === 'ready'"
+              mat-raised-button
+              color="primary"
+              [disabled]="!dirty"
```

Comment:
When the "Save changes" button is disabled, it doesn't look great given the background colour:

![image](https://user-images.githubusercontent.com/3219480/215473037-ef66af52-eba5-4dbe-a563-cee69689865d.png)

I think it would look better if the button was just hidden completely with `*ngif` than disabled.

Reply:
Hiding it squished the green bar so it's opactity 0 when disabled now